shell/platform/tizen/tizen_view_elementary.h

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_TIZEN_VIEW_ELEMENTARY_H_
+#define EMBEDDER_TIZEN_VIEW_ELEMENTARY_H_
+
+#include "flutter/shell/platform/tizen/tizen_view.h"
+
+#define EFL_BETA_API_SUPPORT
+#include <Ecore.h>
+#include <Elementary.h>
+
+#include <unordered_map>
```

Comment:
```suggestion
#define EFL_BETA_API_SUPPORT
#include <Ecore.h>
#include <Elementary.h>

#include <unordered_map>

#include "flutter/shell/platform/tizen/tizen_view.h"
```

Also applies to `tizen_window_*.h`.

Reply:
I modified the code according to your advice and style guide.
Could you please check again that my understanding is correct?